src/oscontainer.py

```diff
@@ -26,6 +26,10 @@
 # https://access.redhat.com/documentation/en-us/openshift_container_platform/4.1/html/builds/custom-builds-buildah
 NESTED_BUILD_ARGS = ['--storage-driver', 'vfs']
 
+# These packages have their versions as labels
+IMPORTANT_PACKAGES = ['kernel', 'kernel-rt-core', 'systemd', 'ignition', 'ostree',
+                      'rpm-ostree', 'runc', 'strace']
```

Comment:
Why `strace`?
I feel like this should be part of rpm-ostree, something like:
```
packages-primary:
  - kernel kernel-rt-core systemd ignition ostree rpm-ostree podman
```
in the manifest or so and then it'd end up in the commit metadata.

Reply:
Whoops, `strace` was just for testing. Dropped now.

>  I feel like this should be part of rpm-ostree

Hmm, but the commit metadata already contains the full RPM package list.

I think the notion of "important packages" is subjective and should be moved as far downstream as possible so it's in the right context. These labels are more or less a hack to help humans inspecting the oscontainer. Ideally we shouldn't have these labels at all and there'd be a cleaner way to inspect the full rpmdb.